test/types/types.go

```diff
@@ -402,3 +402,18 @@ func GetReplicaAttachments(persistentVolume *v1.PersistentVolume, client clients
 	}
 	return azVolumeAttachmentsReplica, nil
 }
+
+func GetSchedulableNodes(azDiskClient *azDiskClientSet.Clientset, client clientset.Interface) []*v1.Node {
+	nodes := nodeutil.ListAzDriverNodeNames(azDiskClient.DiskV1alpha1().AzDriverNodes(azureconstants.AzureDiskCrdNamespace))
+	var availableNodes []*v1.Node
+
+	for _, nodeName := range nodes {
+		node, err := client.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
```

Comment:
Instead of issuing a `Get` request for each node, `List` all the nodes with `metav1.ListOptions{}` using a `FieldSelector` for the `Spec.Unschedulable` field.